docs/index.html

```diff
@@ -32,7 +32,37 @@
       noCompileLinks: [
         'benchmarks/.*'
       ],
-      relativePath: true
+      relativePath: true,
+      markdown: {
+        renderer: {
+          // Mimic markedjs/marked behavior just modify href - https://github.com/markedjs/marked/blob/master/src/Renderer.ts#L178-L191
+          link(href, title, text) {
+            const originalHref = href;
+            // Check for /docs/docs/ in the href and remove duplication it if present
+            href = href.includes('/docs/docs/') ? href.replace('/docs/', '/') : href;
+
+            // ignore paths /docs/api/ and /docs/best-practices/ in /docs/docs directory
+            const ignorePaths = ['/docs/api', '/docs/best-practices'];
+
+            // Check for /docs/ in the href and remove it if present
+            // Want to ignore paths that are in the ignorePaths array
+            if (!ignorePaths.some(path => href.includes(path))) {
+              href = href.includes('/docs/') ? href.replace('/docs/', '/') : href;
+            }
```

Comment:
```suggestion
            // Check for /docs/ in the href and remove it if present
            if (href.startsWith('/docs/')) {
              // ignore paths /docs/api/ and /docs/best-practices/ in /docs/docs directory
              if (/\/docs\/(?:api|best-practices)(?:\/|$)/.test(href) === false) {
              href = href.slice(5);
              }
            }
```

Reply:
Good call out, updated the checks to reflect the suggestion